app/plugin_install.go

```diff
@@ -212,24 +232,37 @@ func (a *App) removePluginLocally(id string) *model.AppError {
 		return model.NewAppError("removePlugin", "app.plugin.not_installed.app_error", nil, "", http.StatusBadRequest)
 	}
 
-	if pluginsEnvironment.IsActive(id) && manifest.HasClient() {
-		message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_PLUGIN_DISABLED, "", "", "", nil)
-		message.Add("manifest", manifest.ClientManifest())
-		a.Publish(message)
-	}
-
 	pluginsEnvironment.Deactivate(id)
 	pluginsEnvironment.RemovePlugin(id)
 	a.UnregisterPluginCommands(id)
 
-	err = os.RemoveAll(pluginPath)
-	if err != nil {
+	if err := os.RemoveAll(pluginPath); err != nil {
 		return model.NewAppError("removePlugin", "app.plugin.remove.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
 	return nil
 }
 
+func (a *App) notifyPluginEvents(manifest *model.Manifest) error {
```

Comment:
It's not entirely clear what `notifyPluginEvents` means, as a name -- there are so many "notify*plugin" invocations now.

Reply:
removed